### PR TITLE
feat: pictograms-march-01

### PR DESCRIPTION
**Pull request by dudley-ibm**
24 net new pictograms in Data Analytics category
updated categories.yml file 
updated pictogram-master.ai file
note: did not add dashes to name, may have to be added
many more additions to come over the next few days in a separate pr 

#### Changelog

**New**

- word cloud.svg
- 3D chart.svg
- area chart.svg
- box plot.svg
- bubble chart.svg
- candlestick chart.svg
- circling packing.svg
- error bar chart.svg
- evaluation chart.svg
- heat map 02.svg
- high low chart.svg
- histogram chart.svg
- math curve.svg
- multi line chart.svg
- parallel chart.svg
- population diagram.svg
- q-q plot.svg
- radar chart.svg
- relationship diagram.svg
- river chart.svg
- scatter matrix.svg
- t-SNE chart.svg
- tree diagram.svg
- tree map.svg

### CODE REVIEW

**Comment by laurenmrice** [comment]
These pictograms will need the dash naming structure. @emyarod do we need to do this now or can we do this after merging this pr.

**Comment by joshblack** [comment]
@laurenmrice would be great to get the dashes in now 👍

File: packages/pictograms/svg/3D chart.svg [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (emyarod) [inline packages/pictograms/svg/3D chart.svg]: I don't think we can have the file name begin with a number, it looks like that's what's causing the CI errors

**Comment by emyarod** [comment]
I think it needs to happen in this PR @laurenmrice, also not sure if `pictograms.yml` should be updated as well

**Comment by emyarod** [comment]
the CI issues are caused by invalid module names ref #5724

**Comment by laurenmrice** [comment]
-Going to change the naming convention of `pictograms` to match what we have for `icons`:
(-) for spaces
(--) for variants

-Will make updates to the new pictograms here so we can get this merged, and will do the rest of our existing pictograms separately.

**Comment by laurenmrice** [comment]
Have been looking into this today, for the naming convention. For chart pictograms I think we should lead with `chart--variant`, since there are so many of them. This is also the same way we name our current chart pictograms. Let me know if you are okay with that @dudley-ibm 
Screen Shot 2020-03-31 at 1 25 23 PM

**Comment by dudley-ibm** [comment]
That sounds perfect @laurenmrice

**Review by joshblack — APPROVED** [review]

**Review by laurenmrice — APPROVED** [review]

**Review by emyarod — APPROVED** [review]
looks good to me

**Pull request merged**